frontend/dispatcher.lua

```diff
@@ -49,6 +49,7 @@ local settingsList = {
     history = { category="none", event="ShowHist", title=_("History"), device=true,},
     open_previous_document = { category="none", event="OpenLastDoc", title=_("Open previous document"), device=true,},
     filemanager = { category="none", event="Home", title=_("File browser"), device=true,},
+    show_terminal = { category = "none", event = "TerminalStart", title = 'Show terminal', device = true, },
```

Comment:
`title = 'Show terminal'` doesn't allow for translation, it should be `title = _('Show terminal')` instead, and no spaces between assignments inside table, to be consequential ;-). Otherwise lgtm.

Reply:
And may be move them (with `edit_last_edited_file` from #6600), later in the list and the order, after Wallabag/calibre, in a separator section of their own (these are plugins, and aren't much related to reading like the first entries are).

Reply:
What they said, and thanks for the PR! :-)